tests/tests.js

```diff
@@ -79,9 +79,13 @@ var runTests = function runTests(arrayFrom) {
 
 	test('it includes Object.prototype values when it is polluted', function (t) {
 		/* eslint-disable no-extend-native */
-		Object.prototype[1] = 42;
```

Comment:
Absolutely! That's part of why this is done in tests - to make sure that if someone *else* does it, this library doesn't break.

Additionally, this tests that the lookup is done up the prototype chain, and not with an own property (as your original modification covered as well).